into/backends/ssh.py

```diff
@@ -206,7 +206,7 @@ def file_to_temp_ssh_file(typ, data, **kwargs):
     """ Generic convert function sending data to ssh(data)
 
     Needs to be partially evaluated with a type"""
-    fn = '.%s.%s' % (uuid.uuid1(), typ.canonical_extension)
+    fn = '%s.%s' % (uuid.uuid1(), typ.canonical_extension)
```

Comment:
is per the conversation with you and @quasiben? maybe a small comment stating why we can't hide this file.